app/settings/surveys/surveys.controller.js

```diff
@@ -49,7 +49,7 @@ function (
     };
 
     $scope.deleteSurvey = function (survey) {
-        Notify.confirmDelete('notify.form.delete_form_confirm', 'notify.form.delete_form_confirm_desc').then(function () {
+        Notify.deleteWithInput('survey', survey.name).then(function () {
```

Comment:
I really like this being introduced as a general pattern, nice job

Reply:
Yes, I think its easier to understand whats going on this way...